packages/odyssey-storybook/src/components/odyssey-mui/Table/Table.stories.tsx

```diff
@@ -0,0 +1,472 @@
+import TableBody from "@mui/material/TableBody";
+import TableCell from "@mui/material/TableCell";
+import TableContainer from "@mui/material/TableContainer";
+import TableHead from "@mui/material/TableHead";
+import TableRow from "@mui/material/TableRow";
+import TableSortLabel from "@mui/material/TableSortLabel";
+import Checkbox from "@mui/material/Checkbox";
+import { MuiThemeDecorator } from "../../../../.storybook/components";
+import TableMdx from "./Table.mdx";
+
+export default {
+  title: `MUI Components/Table`,
+  component: Table,
+  parameters: {
+    docs: {
+      page: TableMdx,
+    },
+  },
+  argTypes: {
+    disabled: {
+      control: "boolean",
+      defaultValue: false,
+    },
+    icon: {
+      control: "text",
+      defaultValue: null,
+    },
+    label: {
+      control: "text",
+      defaultValue: "Asteroids",
+    },
+    wrapped: {
+      control: "boolean",
+      defaultValue: false,
+    },
+  },
+  decorators: [MuiThemeDecorator],
+};
+
+interface Data {
+  name: string;
+  radius: number;
+  type: string;
+  perihelion: string;
+  descriptor: string;
+}
+
+function createData(
+  name: string,
+  radius: number,
+  type: string,
+  perihelion: string,
+  descriptor: string
+): Data {
+  return {
+    name,
+    radius,
+    type,
+    perihelion,
+    descriptor,
+  };
+}
+
+const rows = [
+  createData("Mercury", 2440, "Terrestrial", "--", "Mercurian"),
+  createData("Venus", 6051, "Terrestrial", "--", "Venusian"),
+  createData("Earth", 6378, "Terrestrial", "January 4, 2023", "Terran"),
+  createData("Mars", 3396, "Terrestrial", "June 21, 2022", "Martian"),
+  createData("Jupiter", 71492, "Gas giant", "January 21, 2023", "Jovian"),
+  createData("Saturn", 60268, "Gas giant", "November 29, 2032", "Saturnian"),
+  createData("Uranus", 25559, "Ice giant", "August 17-19, 2050", "Uranian"),
+  createData("Neptune", 24764, "Ice giant", "September 4, 2042", "Neptunian"),
+];
+
+function descendingComparator<T>(a: T, b: T, orderBy: keyof T) {
+  if (b[orderBy] < a[orderBy]) {
+    return -1;
+  }
+  if (b[orderBy] > a[orderBy]) {
+    return 1;
+  }
+  return 0;
+}
+
+type Order = "asc" | "desc";
+
+function getComparator<Key extends keyof any>(
+  order: Order,
+  orderBy: Key
+): (
+  a: { [key in Key]: number | string },
+  b: { [key in Key]: number | string }
+) => number {
+  return order === "desc"
+    ? (a, b) => descendingComparator(a, b, orderBy)
+    : (a, b) => -descendingComparator(a, b, orderBy);
+}
+
+// This method is created for cross-browser compatibility, if you don't
+// need to support IE11, you can use Array.prototype.sort() directly
+function stableSort<T>(
+  array: readonly T[],
+  comparator: (a: T, b: T) => number
+) {
+  const stabilizedThis = array.map((el, index) => [el, index] as [T, number]);
+  stabilizedThis.sort((a, b) => {
+    const order = comparator(a[0], b[0]);
+    if (order !== 0) {
+      return order;
+    }
+    return a[1] - b[1];
+  });
```

Comment:
This can use our ascending comparator right?

Reply:
Oh, just noticed this is in a compatibility area.

Reply:
I tried, but the sorting got borked -- not sure why. This is another thing I'm cool tearing out once we aren't expecting parity with the MUI examples.